ts/nni_manager/extensions/experiments_manager/index.ts

```diff
@@ -1,4 +1,24 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-export { NNIExperimentsManager } from './manager';
+import { ExperimentsManager } from './manager';
+export { ExperimentsManager } from './manager';
+
+let singleton: ExperimentsManager | null = null;
```

Comment:
Is this a real singleton?

Reply:
Yes, each NNI manager process will have only one experiments manager.

Reply:
It's not exported, so the only way to access it is `getExperimentManager()`.